server.js

```diff
@@ -19,6 +33,56 @@ app.get('/', (req, res) => {
     }
 })
 
+//create user in database from request data
+app.post('/signUp', (req, res) => {
+    //create user object
+    //storing the password as plain text is bad for security, but i'm going to do it anyway
+    let user = {}
+    user.userName = req.body.user;
+    user.password = req.body.pass;
+    user.email = req.body.email;
+    user.authenticated = 0;
+    user.authenticateTime = '';
+    userModel.find({'userName': req.body.user}, function(err, data){
+        if(err) return
+        if(data.length===0){
+            userModel.create(user)
+            res.status(200).redirect("/")
+        }
+        else{
+            res.status(200).send("User already exists");
+        }
+    })
+    console.log(user);
+    
+})
+
+//finds user by username/email and checks password against what is stored in db, responds with user token, pass this token into other requests to ensure user is logged in
+app.post('/login', (req, res) => {
+    //console.log(req.body);
+    let userName = req.body.user;
+    let pw = req.body.pass;
+    const now = new Date()
+    //do validation here
+    //lookup user by userName
+    userModel.findOne({'userName':userName},function(err, user){
+        if (err) return
+        var id = user._id
```

Comment:
Don't use var. Use let if you need to. Var's scope is the entire program, so use let instead.